### ISSUE TITLE
Permissions Delete instead of DeleteChild

### ISSUE THREAD

**Issue by bastienperez** [canix1/ADACLScanner#53]
Hello,
I found an issue and don't understand why.
If I set an object (ie an user) protected from deletion, the parent object gets ACE `Deny` with `DeleteChild` for `Everyone`.
```powershell
Get-ACL -path "AD:$object" | select -ExpandProperty Access | ?{ $_.IdentityReference -eq "Everyone" }
ActiveDirectoryRights : DeleteChild
InheritanceType       : None
ObjectType            : 00000000-0000-0000-0000-000000000000
InheritedObjectType   : 00000000-0000-0000-0000-000000000000
ObjectFlags           : None
AccessControlType     : Deny
IdentityReference     : Everyone
IsInherited           : False
InheritanceFlags      : None
PropagationFlags      : None
```

But, with ADACLScan, I get the permissions `Delete` instead of `DeleteChild`:
```powershell
Object            : xxx
ObjectClass       : organizationalUnit
IdentityReference : S-1-1-0
Trustee           : Everyone
Access            : Deny
Inherited         : False
Apply To          : This Object Only
Permission        : Delete
````

**Comment by canix1** [comment]
Hi, 

I agree that delete is a simplified output that could mislead.
I will fix that.
Until then, you can use -RAW flag that will produce the same access but without translation of schemaGuids's.

**Comment by canix1** [comment]
This was fixed in 7.8

**Comment by bastienperez** [comment]
Awesome, thanks!
